samples/c/hello_classification_ov/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+ie_add_sample(NAME hello_classification_ov_c
```

Comment:
Why do we need to have an additional sample?
Can we change old sample application?

Reply:
yes, we can do it similar with benchmark_legacy for old sample application. We are planning to do it in 2022.3.

Reply:
When we introduced OV 2.0 API we moved all cpp samples to new API. Because we don't want to demonstrate legacy API.
Here I hope we should use the same strategy and don't demonstrate legacy API

Reply:
Agree!